components/styled-links.js

```diff
@@ -12,7 +12,6 @@ const basicLinkStyles = {cursor: 'pointer'}
 const anchorStyles = {
   textDecoration: 'underline',
   color: colors.primaryMed,
-  ':visited': {color: colors.secondary},
```

Comment:
I love how a one line change can make such a huge impact on the code base. I also love that snapshot testing shows us what is impacted by the change!